src/blueprint/azext_blueprint/_help.py

```diff
@@ -0,0 +1,451 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['blueprint'] = """
+    type: group
+    short-summary: Commands to manage blueprint.
+"""
+
+helps['blueprint create'] = """
+    type: command
+    short-summary: Create a blueprint definition.
+    examples:
+      - name: SubscriptionBlueprint
+        text: |-
+               az blueprint create --subscription "00000000-0000-0000-0000-000000000000" --name \\
+               "simpleBlueprint" --description \\
+               "blueprint contains all artifact" --target-scope \\
+               "subscription"
+      - name: ManagementGroupBlueprint
+        text: |-
+               az blueprint create --management-group myManagementGroup --name \\
+               "simpleBlueprint" --description \\
+               "blueprint contains all artifact" --target-scope \\
+               "subscription"
+"""
+
+helps['blueprint update'] = """
+    type: command
+    short-summary: Update a blueprint definition.
+"""
+
+helps['blueprint delete'] = """
+    type: command
+    short-summary: Delete a blueprint definition.
+    examples:
+      - name: ManagementGroupBlueprint
+        text: |-
+               az blueprint delete --management-group myManagementGroup --name \\
+               "simpleBlueprint"

```

Comment:
is there consisntent resource naming convention in help? Would suggest to have one. such as myResource etc

Reply:
For common resource names, we can do that. For specific resource names, they may have naming constraints, I usually do not modify such values from swagger example.

Reply:
got it. While Swaggers themselves aren't consistent at all. Let's align in CLI